admin/js/svg-map-by-saedi-admin.js

```diff
@@ -31,52 +31,54 @@
 	// svg.selectAll("circle").enter().append("circle").on("click", function(d) {
 	// 	console.log("clicked!");
 	// });
```

Comment:
I ran phpcs throughout the plugin folder. It picked js file too! I thought it would only pick php files. Anyways @smjrifle have a look at it.